src/IconButton/index.scss

```
@@ -0,0 +1,58 @@
+.nds-icon-button {
+  background: none;
+  color: inherit;
+  border: none;
+  font: inherit;
+  cursor: pointer;
+  outline: inherit;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: var(--button-radius);
+  height: 32px;
+  width: 32px;
+
+  &.fontSize--l {
+    [class^="narmi-icon-"] {
+      transform: translate(0px, -1px);
+    }
+  }
+}
+
+.nds-icon-button--action {
+  --button-radius: 50%;
+  color: var(--color-secondary);
```

Initial color should be secondary text color:

```suggestion
  color: var(--font-color-secondary);
```